openstates/wi/bills.py

```diff
@@ -123,12 +123,13 @@ def scrape_bill_list(self, chamber, session, url):
             return
         doc = lxml.html.fromstring(data)
         doc.make_links_absolute(url)
-        links = doc.xpath('//li//a')
-        for link in links:
-            bill_url = link.get('href')
+        bill_list = doc.xpath('//ul[@class="infoLinks"]/li/div[@class="row-fluid"]')
+        for b in bill_list:
+            bill_url = b.xpath('./div[@class="span3"]/a/@href')[0]
             bill_id = bill_url.rsplit('/', 1)[-1]
+            print bill_id
```

Comment:
We tend to avoid leaving prints in the source (clutters it up a bit) in favor of `self.debug` or `self.info`